src/ucp/proto/ucp_context.c

```diff
@@ -111,6 +111,12 @@ static int ucp_is_resource_enabled(uct_tl_resource_desc_t *resource,
         }
     }
 
+    /* Disable the mmap and xpmem 'devices'. ONLY for now - use sysv for mm .
+     * This will be removed after multi-rail is supported */
+    if (!strcmp(resource->dev_name,"mmap") || !strcmp(resource->dev_name, "xpmem")) {
+        device_enabled  = 0;
+    }
+
```

Comment:
This is a temporary workaround. Any solution to hanlde multiple devices until it is actualy supported in ucp, would be temporary. This is the simplest for now.